Source/BoundaryConditions/PML.cpp

```diff
@@ -1298,16 +1298,16 @@ PML::PushPSATD (ablastr::fields::MultiFabRegister& fields, const int lev)
 {
     ablastr::fields::VectorField pml_E_fp = fields.get_alldirs(FieldType::pml_E_fp, lev);
     ablastr::fields::VectorField pml_B_fp = fields.get_alldirs(FieldType::pml_B_fp, lev);
-    ablastr::fields::ScalarField pml_F_fp = fields.get(FieldType::pml_F_fp, lev);
-    ablastr::fields::ScalarField pml_G_fp = fields.get(FieldType::pml_G_fp, lev);
+    ablastr::fields::ScalarField pml_F_fp = (fields.has(FieldType::pml_F_fp, lev)) ? fields.get(FieldType::pml_F_fp, lev) : nullptr;
```

Comment:
hehe, that's one way to quickly get the behavior in without changing downstream :D okok, nice idea and we can make this more robust later.